CMakeLists.txt

```diff
@@ -81,7 +87,7 @@ add_library(fv3atm
   fv3_cap.F90
   module_fv3_config.F90
   module_fcst_grid_comp.F90
-  stochastic_physics/stochastic_physics_wrapper.F90
+  stochastic_physics_wrapper/stochastic_physics_wrapper.F90
```

Comment:
What is this change used for? I don't have stochastic_physics_wrapper/stochastic_physics_wrapper.F90 under fv3atm.

Reply:
I'm not sure why I made that change, so I just reverted it.